css/tideline-svg.less

```diff
@@ -63,6 +63,38 @@ g.svg-tooltip-smbg {
   }
 }
 
+g.svg-tooltip-range {
+  polygon {
+    fill: @fill-lightest;
+    opacity: 0.7;
+    stroke-width: 1.5;
+    stroke-opacity: 0.4;
+    stroke: @range;
+  }
+}
+
+.smbgRangeInfo {
+  rect {
+    stroke-width: 1.5;
+    stroke-opacity: 0.4;
+    stroke: @range;
+  }
+
+  text {
+    &.label {
+      text-anchor: start;
+      font-size: 12px;
+      fill: #a6a6a6;
```

Comment:
Can you create a variable for this color in tideline-colors? I think we've been doing a good job of keeping all colors in one place. (New Basics stuff excepted, for now.)